src/Network/Network/help/New-AzNetworkManagerSecurityUserRule.md

```diff
@@ -0,0 +1,394 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://learn.microsoft.com/powershell/module/az.network/new-aznetworkmanagersecurityuserrule
+schema: 2.0.0
+---
+
+# New-AzNetworkManagerSecurityUserRule
+
+## SYNOPSIS
+Creates a security user rule.
+
+## SYNTAX
+
+### Custom (Default)
+```
+New-AzNetworkManagerSecurityUserRule -Name <String> -RuleCollectionName <String>
+ -SecurityUserConfigurationName <String> -NetworkManagerName <String> -ResourceGroupName <String>
+ [-Description <String>] -Protocol <String> -Direction <String>
+ [-SourceAddressPrefix <PSNetworkManagerAddressPrefixItem[]>]
+ [-DestinationAddressPrefix <PSNetworkManagerAddressPrefixItem[]>] [-SourcePortRange <String[]>]
```

Comment:
confirm required params here?

Reply:
Currently in the backend either source/dest address prefix or source/port needs to be specified which I think is not a correct validation. From portal these params are optional.